.evergreen.yml

```diff
@@ -220,104 +229,229 @@ tasks:
     commands:
       - func: checkout
       - func: install
+        vars:
+          node_js_version: "14.15.1"
       - func: check
+        vars:
+          node_js_version: "14.15.1"
   - name: check_win
     commands:
       - func: checkout
       - func: install_win
+        vars:
+          node_js_version: "14.15.1"
       - func: check_win
-  - name: test_m40x
+        vars:
+          node_js_version: "14.15.1"
+  - name: test_m40x_n12
+    commands:
+      - func: checkout
+      - func: install
+        vars:
+          node_js_version: "12.20.0"
+      - func: test
+        vars:
+          mongosh_server_test_version: "4.0.x"
+          node_js_version: "12.20.0"
+          mongosh_skip_version_check: "i-will-not-complain-if-mongosh-does-not-work"
+  - name: test_m42x_n12
+    commands:
+      - func: checkout
+      - func: install
+        vars:
+          node_js_version: "12.20.0"
+      - func: test
+        vars:
+          mongosh_server_test_version: "4.2.x"
+          node_js_version: "12.20.0"
+          mongosh_skip_version_check: "i-will-not-complain-if-mongosh-does-not-work"
+  - name: test_m44x_n12
+    commands:
+      - func: checkout
+      - func: install
+        vars:
+          node_js_version: "12.20.0"
+      - func: test
+        vars:
+          mongosh_server_test_version: "4.4.x"
+          node_js_version: "12.20.0"
+          mongosh_skip_version_check: "i-will-not-complain-if-mongosh-does-not-work"
+  - name: test_m40x_n12_win
+    commands:
+      - func: checkout
+      - func: install_win
+        vars:
+          node_js_version: "12.20.0"
+      - func: test_win
+        vars:
+          mongosh_server_test_version: "4.0.x"
```

Comment:
Isn't it clearer to use all caps for environment vars?

Reply:
So… these are not environment variables per se, but evergreen expansions, which are usually all lowercase, and it’s only the import-expansions script that turns them to uppercase

So I’d probably like to be consistent between `node_js_version` and `mongosh_server_test_version`, but I guess if we wanted to we could turn them all uppercase

Reply:
Oh no then is great, so are just interpolated strings, makes sense